src/prefect/cli/flow_run.py

```diff
@@ -113,3 +115,23 @@ async def delete(id: UUID):
             exit_with_error(f"Flow run '{id}' not found!")
 
     exit_with_success(f"Successfully deleted flow run '{id}'.")
+
+
+@flow_run_app.command()
+async def cancel(id: UUID):
+    """Cancel a flow fun by ID."""
+    async with get_client() as client:
+        cancelling_state = State(type=StateType.CANCELLED, name="Cancelling")
+        try:
+            result = await client.set_flow_run_state(
+                flow_run_id=id, state=cancelling_state
+            )
+        except ObjectNotFound as exc:
+            exit_with_error(f"Flow run '{id}' not found!")
+
+    if result.status == SetStateStatus.ABORT:
+        exit_with_error(
+            f"Flow run '{id}' was unable to be cancelled. Reason: '{result.details.reason}'"
```

Comment:
Instead of wrapping things in quotes manually you can, generally, use `!r` in the format string, it can read a bit nicer:

```suggestion
            f"Flow run {id!r} was unable to be cancelled. Reason: {result.details.reason!r}"
```

Reply:
Awesome - thanks!